src/features/layers_in_area/readme.md

```diff
@@ -0,0 +1,37 @@
+## Layer in area
+
+This feature manage all layers that intersecting area of interest (a.k.a focused geometry).
+Every time when geometry changes - this feature:
```

Comment:
probably needs more clarification what affects Layers in area

Reply:
```suggestion
Every time when focused geometry changes - this feature:
```

Reply:
good point, i will use vkozio variant